Foundation/Process.swift

```diff
@@ -102,6 +102,62 @@ private func processIsEqual(_ a : UnsafeRawPointer?, _ b : UnsafeRawPointer?) ->
     return true
 }
 
+#if os(Windows)
+private func quoteWindowsCommandArg(arg: String) -> String {
```

Comment:
Please remove the trailing semicolons from within this function

Reply:
Oof, that's what I get for copying cpp code without thinking about it.